charts/datadog/values.yaml

```diff
@@ -141,14 +141,14 @@ datadog:
 
     # datadog.dogstatsd.nonLocalTraffic -- Enable this to make each node accept non-local statsd traffic
     ## ref: https://github.com/DataDog/docker-dd-agent#environment-variables
-    nonLocalTraffic: false
+    nonLocalTraffic: true
```

Comment:
this is set to false on purpose for security reason: mainly because if a node is not properly protected by a firewall, anyone can push traces to the agent.

Reply:
Not sure as `useHostPort: false` by default